‘DOC-2652-gsql-files-output-4.2’ (#335)
tigergraph/gsql-docs: modules/querying/pages/data-types.adoc

> When referring to a `FILE` object, we always capitalize the word `FILE` to distinguish it from ordinary files.

Reviewer: ```suggestion
When referring to a `FILE` object, this documentation capitalizes the word `FILE` to distinguish it from ordinary files.
```
Reword to clarify that the capitalization is our own style rule, not a syntax rule.
